config/locales/cy/activemodel.yml

```diff
@@ -3,19 +3,465 @@ cy:
   accessibility:
     problem_text: melborp a si erehT
   activemodel:
+    attributes:
```

Comment:
Do we need the extra translations? Only the citizen pages are being served in dual languages.